deploy/01_deploy.js

```diff
@@ -0,0 +1,36 @@
+module.exports = async ({
+    getNamedAccounts,
+    deployments,
+    getChainId,
+    getUnnamedAccounts,
+  }) => {
+    const {deploy} = deployments;
+    const {deployer, foundationInitialAddress} = await getNamedAccounts();
+
+    console.log(foundationInitialAddress)
+  
+    console.log("----------------------------------------------------")
+    console.log('Deploying BTRUST');
+    const btrust = await deploy('BTRUST', {
+      from: deployer,
+      args: [foundationInitialAddress],
+    });
+    console.log("BTRUST deployed to: ", btrust.address)
+
+    console.log("----------------------------------------------------")
+    console.log('Deploying Timelock');
+    const timelock = await deploy('Timelock', {
+      from: deployer,
+      args: [foundationInitialAddress, 7 * 24 * 60 * 60],
```

Comment:
7 * 24 * 60 * 60 -> add to config plz

Reply:
Okay. Also with the new deploy script and hardhat config, I don't think we need btrust.config.js file, is it fine to get rid of it?